llama_stack/apis/post_training/post_training.py

```diff
@@ -182,7 +182,19 @@ async def supervised_fine_tune(
         ),
         checkpoint_dir: str | None = None,
         algorithm_config: AlgorithmConfig | None = None,
-    ) -> PostTrainingJob: ...
+    ) -> PostTrainingJob:
+        """Supervised fine-tune a model
```

Comment:
Doesn't sound like proper English to me. Perhaps "Run supervised fine-tuning of a model"?